### ISSUE TITLE
App crashes every time on startup - unknown color - dark mode enabled

### ISSUE THREAD

**Issue by elpraga** [stefan-niedermann/nextcloud-deck#514]
```
App Version: 1.4.2
App Version Code: 1004002
App Flavor: play

Files App Version Code: 30110190

---

OS Version: 4.4.153-perf+(V11.0.8.0.PFGEUXM)
OS API Level: 28
Device: lavender
Manufacturer: Xiaomi
Model (and Product): Redmi Note 7 (lavender_eea)

---

java.lang.IllegalArgumentException: Unknown color
	at android.graphics.Color.parseColor(Color.java:1391)
	at it.niedermann.nextcloud.deck.util.ViewUtil.getTintedImageView(ViewUtil.java:68)
	at it.niedermann.nextcloud.deck.util.DrawerMenuUtil.inflateBoards(DrawerMenuUtil.java:116)
	at it.niedermann.nextcloud.deck.ui.MainActivity.inflateBoardMenu(MainActivity.java:655)
	at it.niedermann.nextcloud.deck.ui.MainActivity.lambda$null$15$MainActivity(MainActivity.java:492)
	at it.niedermann.nextcloud.deck.ui.-$$Lambda$MainActivity$7hz4rYX5wlxgwDk4_nzEJFw4z30.onChanged(Unknown Source:4)
	at androidx.lifecycle.LiveData.considerNotify(LiveData.java:131)
	at androidx.lifecycle.LiveData.dispatchingValue(LiveData.java:149)
	at androidx.lifecycle.LiveData.setValue(LiveData.java:307)
	at androidx.lifecycle.MutableLiveData.setValue(MutableLiveData.java:50)
	at androidx.lifecycle.Transformations$3.onChanged(Transformations.java:188)
	at androidx.lifecycle.MediatorLiveData$Source.onChanged(MediatorLiveData.java:152)
	at androidx.lifecycle.LiveData.considerNotify(LiveData.java:131)
	at androidx.lifecycle.LiveData.dispatchingValue(LiveData.java:149)
	at androidx.lifecycle.LiveData.setValue(LiveData.java:307)
	at androidx.lifecycle.MutableLiveData.setValue(MutableLiveData.java:50)
	at androidx.lifecycle.LiveData$1.run(LiveData.java:91)
	at android.os.Handler.handleCallback(Handler.java:873)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:201)
	at android.app.ActivityThread.main(ActivityThread.java:6810)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:547)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:873)

```

**Comment by stefan-niedermann** [comment]
Thank you for the report! Working on a fix right now.

**Comment by stefan-niedermann** [comment]
I have fixed the problem in version **`1.4.3`**. Because of corona the distribution at Play Store might take **up to 2 weeks**.

Thank you again for reporting this issue :slightly_smiling_face: This kind of feedback is incredible valuable for us to stabilize the app for various different envrionments. If you need some free vouchers for the app (e.g. for coworkers or family) send an e-mail with a link to this issue to info@niedermann.it

Regards
Stefan
